src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -753,6 +753,25 @@ class spell_mage_firestarter_dots : public AuraScript
     }
 };
 
+// 108853 - Fire Blast
+class spell_mage_fire_blast : public SpellScript
+{
+    bool Load() override
+    {
+        return GetCaster()->GetTypeId() == TYPEID_PLAYER;
```

Comment:
IsPlayer preferably to object typeId

Reply:
Honestly completely unneccessary check

Reply:
deleted

Reply:
well, that is true, who cares if it's a player to set it to guarateed crit